contribs/gmf/src/services/objecteditingquery.js

```diff
@@ -220,3 +220,17 @@ gmf.ObjectEditingQuery.GetQueryableLayerNodesResponse;
  * }}
  */
 gmf.ObjectEditingQuery.QueryableLayerInfo;
+
```

Comment:
Well, as it turns out it does!  `make check` worked after I rebased today and I was able to put a live demo.  Look at: https://adube.github.io/ngeo/oe-layers-list-undefined/examples/contribs/gmf/objecteditinghub.html, then run the example or hosted app.  You'll see that the list is now working.

It exactly as I predicted: the compiler was renaming the inner properties of the typedef because they were not exported.  In the templates, the code is not minimized so it was using the original name, not the name 'renamed' by the compiler.  So yes: putting `@export` did work.